src/ui/windows/TogglDesktop/TogglDesktop/ui/Converters/BytesToStringConverter.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace TogglDesktop.Converters
+{
+    public class BytesToStringConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var byteCount = (long)value;
+            var humanReadableBytes = BytesToString(byteCount);
+            return humanReadableBytes;
+        }
+
+        private static string BytesToString(long byteCount)
+        {
+            string[] suf = { "B", "KB", "MB", "GB", "TB", "PB", "EB" };
+            if (byteCount == 0)
+                return "0" + suf[0];
+            var bytes = Math.Abs(byteCount);

```

Comment:
Why not add a `using static System.Math` and simplify it all to:
```cs
var bytes = Abs(byteCount);
var place = Convert.ToInt32(Floor(Log(bytes, 1024)));
var num = Round(bytes / Pow(1024, place), 1);
return $"{Sign(byteCount) * num} {suf[place]}";
```
(+string interpolation)

Reply:
Completely agree 👍  (except `Convert.ToInt32` thing as it creates ambiguity with the `Convert` method above and causes compilation errors).